configuration_version_test.go

```diff
@@ -144,6 +144,46 @@ func TestConfigurationVersionsRead(t *testing.T) {
 	})
 }
 
+func TestConfigurationVersionsReadWithOptions(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	wTest, wTestCleanup := createWorkspaceWithVCS(t, client, orgTest, true)
+	defer wTestCleanup()
+
+	// Hack: Wait for TFC to ingress the configuration and queue a run
+	time.Sleep(3 * time.Second)
+
+	w, err := client.Workspaces.ReadByIDWithOptions(ctx, wTest.ID, &WorkspaceReadOptions{
+		Include: "current-run.configuration-version",
+	})
+
+	if err != nil {
+		require.NoError(t, err)
+	}
+
+	if w.CurrentRun == nil {
+		t.Fatal("A run was expected to be found on this workspace as a test pre-condition")
+	}
+
+	cv := w.CurrentRun.ConfigurationVersion
+
+	t.Run("when the configuration version exists", func(t *testing.T) {
+		options := &ConfigurationVersionReadOptions{
+			Include: "ingress-attributes",
```

Comment:
Could we use a constant here instead of text? Even though one can determine the [relationships via looking at the API Docs](https://www.terraform.io/docs/cloud/api/configuration-versions.html#show-a-configuration-version-39-s-commit-information), I think it'd be helpful for users of this library to be able to see the list of relationships here, and when they use it, to reference a constant instead of strings.

Reply:
Another point that is not relevant at all to this PR, but its worth mentioning just for awareness. I think having a typed string for all of the `Include` in options would be very helpful for users of this library. What you can include is only a limited set of strings, not all strings, so being explicit about this is better imo.

Reply:
I generally agree, an enumerated list of types would be nice here, but the fact that you can nest values makes it a little wonky and probably not as ergonomic as we'd hope? What would you do with something like the one below with `current-run.configuration-version`? `CurrentRunType + "." + "ConfigurationVersionType`? Starts to feel more overbearing than helpful, at first glance.

Reply:
I can see that. Although, the list of relations for every model is bounded and only has a handful of relations at most. Runs seems like it's on the higher end with 7 relations. `CurrentRunConfigurationVersion` seems good to me?